Replace some strings which ones that (sometimes) make more sense.
Use QT3 version when QT4 is not available (On el5).
Wrap vditool for fixed LD_LIBRARY_PATH.

Lubomir Rintel <lkundrak@v3.sk>

--- VirtualBox-2.0.2/src/VBox/Installer/linux/VBox.sh.setup	2008-09-01 14:23:31.000000000 +0200
+++ VirtualBox-2.0.2/src/VBox/Installer/linux/VBox.sh	2008-09-20 11:01:12.000000000 +0200
@@ -36,19 +36,20 @@
     cat << EOF
 WARNING: The vboxdrv kernel module is not loaded. Either there is no module
          available for the current kernel (`uname -r`) or it failed to
-         load. Please recompile the kernel module and install it by
+         load. Reboot the computer, or insert the kernel module by executing
 
-           sudo /etc/init.d/vboxdrv setup
+           '/sbin/modprobe vboxdrv' (as root)
 
          You will not be able to start VMs until this problem is fixed.
 EOF
 elif [ ! -c /dev/vboxdrv ]; then
     cat << EOF
-WARNING: The character device /dev/vboxdrv does not exist. Try
+WARNING: The character device /dev/vboxdrv does not exist. Try running
 
-           sudo /etc/init.d/vboxdrv restart
+           '/sbin/udevtrigger' (as root)
 
-         and if that is not successful, try to re-install the package.
+         and if that is not successful, then you're in situation that
+         can not possibly happen and there's nothing you can do about it.
 
 	 You will not be able to start VMs until this problem is fixed.
 EOF
@@ -86,7 +87,15 @@
 APP=${APP##/*/}
 case "$APP" in
   VirtualBox)
-    exec "$INSTALL_DIR/VirtualBox" "$@"
+    if [ -e "$INSTALL_DIR/VirtualBox" ]
+    then
+        exec "$INSTALL_DIR/VirtualBox" "$@"
+    else
+        exec "$INSTALL_DIR/VirtualBox3" "$@"
+    fi
+    ;;
+  VirtualBox3)
+    exec "$INSTALL_DIR/VirtualBox3" "$@"
     ;;
   VBoxManage)
     exec "$INSTALL_DIR/VBoxManage" "$@"
@@ -103,6 +112,9 @@
   vboxwebsrv)
     exec "$INSTALL_DIR/vboxwebsrv" "$@"
     ;;
+  vditool)
+    exec "$INSTALL_DIR/vditool" "$@"
+    ;;
   *)
     echo "Unknown application - $APP"
     ;;
--- VirtualBox-2.0.2/src/VBox/VMM/VM.cpp.setup	2008-09-01 14:23:13.000000000 +0200
+++ VirtualBox-2.0.2/src/VBox/VMM/VM.cpp	2008-09-20 10:47:26.000000000 +0200
@@ -258,8 +258,8 @@
 #ifdef RT_OS_LINUX
                     pszError = N_("VirtualBox kernel driver not loaded. The vboxdrv kernel module "
                                   "was either not loaded or /dev/vboxdrv is not set up properly. "
-                                  "Re-setup the kernel module by executing "
-                                  "'/etc/init.d/vboxdrv setup' as root");
+                                  "Reboot the computer, or insert the kernel module by executing "
+                                  "'/sbin/modprobe vboxdrv' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not loaded");
 #endif
@@ -300,8 +300,8 @@
 #ifdef RT_OS_LINUX
                     pszError = N_("VirtualBox kernel driver not installed. The vboxdrv kernel module "
                                   "was either not loaded or /dev/vboxdrv was not created for some "
-                                  "reason. Re-setup the kernel module by executing "
-                                  "'/etc/init.d/vboxdrv setup' as root");
+                                  "reason. Insert the kernel module by executing "
+                                  "'/sbin/modprobe vboxdrv' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not installed");
 #endif
